spec/spec_helper.rb

```diff
@@ -45,6 +45,11 @@ class FullModel
   attribute :web_page, :web_page
 
   has_n_of_attribute :web_pages, :web_page
+  has_n_of_attribute :files, :file
+
+  attr_accessor :api
```

Comment:
Does this make `:api` accessible to all `FullModel` classes for the tests? Is there a way we can limit it's scope to where we need it for the tests only? 

This is a very minor note, focused on minimizing public surface area, feel free to ignore.

Reply:
I think we need to because `FullModel` is used everywhere in test and we needed to recreate the scenario like we are doing in real models.